### PR TITLE
Fix search runs

### PR DESCRIPTION
**Pull request by eliasbenussi**
This PR fixes the integration with Mlflow's cli. It is now possible to search runs using `mflow runs list ...`

### CODE REVIEW

**Comment by acroz** [comment]
Hailey has validated that this works with the latest PRs on faculty merged.

File: mlflow_faculty/trackingstore.py [hunk 1]
```diff
@@ -333,7 +333,9 @@ def get_metric_history(self, run_id, metric_key):
                 for faculty_metric in metric_history
             ]
 
-    def search_runs(self, experiment_ids, search_expressions, run_view_type):
+    def search_runs(
+        self, experiment_ids, search_expressions=None, run_view_type=None
```
Comment (acroz) [inline mlflow_faculty/trackingstore.py]:
```suggestion
        self, experiment_ids, search_filter=None, run_view_type=None
```

Comment (acroz) [reply mlflow_faculty/trackingstore.py]: Also update the docstring below

File: mlflow_faculty/trackingstore.py [hunk 2]
```diff
@@ -358,7 +358,9 @@ def search_runs(self, experiment_ids, search_expressions, run_view_type):
             )
             while True:
                 list_runs_response = self._client.list_runs(
-                    self._project_id, experiment_ids=experiment_ids
+                    self._project_id,
+                    experiment_ids=experiment_ids,
+                    lifecycle_stage=run_view_type,
```
Comment (acroz) [inline mlflow_faculty/trackingstore.py]: Need to map MLflow `ViewType` to faculty `LifecycleStage`.

Comment (haileyfong) [reply mlflow_faculty/trackingstore.py]: We already have this mapping, so I think we can use it.

**Comment by eliasbenussi** [comment]
LGTM

**Pull request merged**